file-formats/sfpf/type/zif_aff_sfpf_v1.intf.abap

```diff
@@ -0,0 +1,40 @@
+INTERFACE zif_aff_sfpf_v1
+  PUBLIC.
+  TYPES:
+    BEGIN OF ty_general_information,
+      "! <p class="shorttext">Dataprovider</p>
+      "! RAP Service Definition that implements the business context for this form template
+      dataprovider    TYPE c LENGTH 40,
+      "! <p class="shorttext">Font Embed</p>
+      "! Automatically embed font files into the output.
+      "! Useful if your output uses fonts that are not delivered by default,
+      "! for asian fonts or to include your own branding.
+      font_embed           TYPE abap_bool,
+      "! <p class="shorttext">Layout Mirroring</p>
+      "! When activated if your form template is designed in an LTR language
+      "! and your target output is an RTL language,
+      "! layout will be automatically mirrored in the output.
+      layout_mirroring       TYPE abap_bool,
+      "! <p class="shorttext">Reduce Data Volume</p>
+      "! Reduce data input volume based on the form design
+      reduce_data_vol       TYPE abap_bool,
```

Comment:
This is just a minor finding. I believed I mentioned it in my last review, but it seems I did not.

Is there any reason you didn't use `reduce_data_volume`?

```suggestion
      reduce_data_volume       TYPE abap_bool,
```

Reply:
The idea was to reduce the size of the property name. But I changed it to display the full text now.